runners/direct-java/pom.xml

```diff
@@ -77,95 +163,6 @@
         </executions>
       </plugin>
 
-      <!-- Ensure that the Maven jar plugin runs before the Maven
-        shade plugin by listing the plugin higher within the file. -->
-      <plugin>
```

Comment:
Not quite true - that is the default phase for `surefire`, but we bind it to `integration-test`. Either way, they have a well-defined ordering w.r.t. `package` so it is a non-issue.